bluebutton-data-model-rif-samples/src/main/resources/rif-static-samples/sample-mct-beneficiaries.txt

```diff
@@ -0,0 +1,9 @@
+INSERT|-201|14|999|46250|01-JAN-1910|2|1|0|0|0|10|0|0|MBP000201A|DOE|JOHN|A|3456789|17-Mar-1981|2019|4|5|6|7|8|A|B|C|D|4|01-Feb-2014|5|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|4|4|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|5|C|C|C|C|C|C|C|C|C|C|C|C|D|D|D|D|D|D|D|D|D|D|D|D|||||0|0|0|0|0|0|0|0|||||||||||||||||||||||||S4802|S4802|S4802|S4802|0|0|0|0|0|0|0|0|086|086|086|086|||||||||0|0|0|0|||||||||C|C|C|C|C|C|C|C|C|C|C|C|01|01|01|01|00|00|00|00|00|00|00|00|00|00|00|00|00|00|00|00|00|00|00|00
+INSERT|-202|42|999|35487|01-JAN-1910|2|1|0|0|0|10|0|0|MBP000202A|SMITH|STEPHEN|E|3456789|17-Mar-1981|2019|4|5|6|7|8|A|B|C|D|4|01-Feb-2014|5|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|4|4|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|5|C|C|C|C|C|C|C|C|C|C|C|C|D|D|D|D|D|D|D|D|D|D|D|D|H0104|H0104|H0104|H0104|0|0|0|0|0|0|0|0|012|012|012|012|||||||||0|0|0|0|||||||||||||0|0|0|0|0|0|0|0|||||||||||||||||||||||||C|C|C|C|C|C|C|C|C|C|C|C|02|02|02|02|00|00|00|00|00|00|00|00|10|10|10|10|00|00|00|00|00|00|00|00
+INSERT|-203|05|999|72653|02-FEB-1910|1|1|0|0|0|10|0|0|MBP000203A|JOHNSON|MICHAEL|V|3456789|17-Mar-1981|2019|4|5|6|7|8|A|B|C|D|4|01-Feb-2014|5|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|4|4|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|5|C|C|C|C|C|C|C|C|C|C|C|C|D|D|D|D|D|D|D|D|D|D|D|D|||||0|0|0|0|0|0|0|0|||||||||||||||||||||||||S4802|S4802|S4802|S4802|0|0|0|0|0|0|0|0|073|073|073|073|||||||||0|0|0|0|||||||||C|C|C|C|C|C|C|C|C|C|C|C|NA|NA|NA|NA|00|00|00|00|00|00|00|00|01|01|01|01|00|00|00|00|00|00|00|00
```

Comment:
PTD_SGMT_***_ID is stored in the production database as '000' whereas you have '0' as the sample data.  I think we should emulate how the data is actually coming to us from CCW so issues don't arise when MCT's code is migrated to production.

Reply:
Great catch: thanks so much! I've fixed this in a new commit.

@dho937 Can you take a second look at it now?